api/src/test/java/org/openmrs/api/UserServiceTest.java

```diff
@@ -473,9 +475,9 @@ public void changePassword_shouldMatchOnIncorrectlyHashedSha1StoredPassword() {
 		executeDataSet(XML_FILENAME);
 		Context.logout();
 		Context.authenticate("incorrectlyhashedSha1", "test");
-
+		Context.addProxyPrivilege(PrivilegeConstants.GET_GLOBAL_PROPERTIES);
 		userService.changePassword("test", "Tester12");
-
+		Context.removeProxyPrivilege(PrivilegeConstants.GET_GLOBAL_PROPERTIES);
```

Comment:
Wait a minute, if we are going to logout in the next line, do we need the removeProxyPrivilege call?

Reply:
I thought we should remove the privilege before the logout just in case there's a problem during the logout process. As this is a test we could remove this if we want.

Reply:
In that case the test would still fail until we fix the logout problem.

Reply:
Doesn't log out remove all privileges?

Reply:
Yes, it does. It clears the ArrayList that contains the privileges. I was just trying to be extra safe by removing the privilege before the logout. I updated the PR.

Reply:
Logout is enough.